three.js/src/threex/threex-armarkercontrols.js

```diff
@@ -1,7 +1,7 @@
 var ARjs = ARjs || {}
 var THREEx = THREEx || {}
 
-ARjs.MarkerControls = THREEx.ArMarkerControls = function (context, object3d, parameters) {
+ARjs.MarkerControls = THREEx.ArMarkerControls = function (context, object3d, camera, parameters) {
```

Comment:
We can find another solution @nicolocarpignoli ?

Reply:
in which sense?

Reply:
in the sense  i need the camera object as function args , but i think i have no choice.